### ISSUE TITLE
Error Class Service\Payment

### ISSUE THREAD

**Issue by marcellofiore** [Adyen/adyen-php-api-library#73]
PHP version => 7.1.20
Library version => "adyen/php-api-library" : "1.5.0"

Hi, I installed the library via composer, and I try to create a payment request on Test with the sample code => but I receive an error : Class 'Service\Payment' not found. What can it be, can someone help me?

$client = new \Adyen\Client();
$client->setApplicationName("Adyen PHP Api Library Example");
$client->setUsername("YOUR USERNAME");
$client->setPassword("YOUR PASSWORD");
$client->setEnvironment(\Adyen\Environment::TEST);
$service = new Service\Payment($client);
$json = '{
      "card": {
        "number": "4111111111111111",
        "expiryMonth": "6",
        "expiryYear": "2016",
        "cvc": "737",
        "holderName": "John Smith"
      },
      "amount": {
        "value": 1500,
        "currency": "EUR"
      },
      "reference": "payment-test",
      "merchantAccount": "YOUR MERCHANT ACCOUNT"
}';

$params = json_decode($json, true);
$result = $service->authorise($params);

![screen shot 2018-09-21 at 17 23 42](https://user-images.githubusercontent.com/22861591/45890371-1bf34400-bdc3-11e8-8dd0-56147e9d5330.png)

**Comment by rikterbeek** [comment]
Hi @marcellofiore,

It looks like the autoloader is not picking up this class.
Did you run it through composer ? 
```
composer require adyen/php-api-library
```

Make sure you include the autoloader:
```
require __DIR__ . '/vendor/autoload.php';
```


Regards,
Rik
Adyen

**Comment by wesweat** [comment]
Hi Guys,

Don't want to hijack this thread but I got a similar question. I am also trying out Adyen testing code, like Marcel uses. In my case i get 

> Fatal error: Class 'Service\Payment' not found

I have searched and I think it has to do with classes not being loaded but I can't find out why?

On my local XAMPP i have a test.php with 
`require __DIR__ . '/vendor/autoload.php';`

and the rest of adyen test/example code


FYI: I have installed with composer:
` composer require adyen/php-api-library
Using version ^1.5 for adyen/php-api-library
./composer.json has been created
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 3 installs, 0 updates, 0 removals
  - Installing psr/log (1.0.2): Loading from cache
  - Installing monolog/monolog (1.24.0): Downloading (100%)         
  - Installing adyen/php-api-library (1.5.3): Downloading (100%)         
monolog/monolog suggests installing aws/aws-sdk-php (Allow sending log messages to AWS services like DynamoDB)
monolog/monolog suggests installing doctrine/couchdb (Allow sending log messages to a CouchDB server)
monolog/monolog suggests installing ext-amqp (Allow sending log messages to an AMQP server (1.0+ required))
monolog/monolog suggests installing ext-mongo (Allow sending log messages to a MongoDB server)
monolog/monolog suggests installing graylog2/gelf-php (Allow sending log messages to a GrayLog2 server)
monolog/monolog suggests installing mongodb/mongodb (Allow sending log messages to a MongoDB server via PHP Driver)
monolog/monolog suggests installing php-amqplib/php-amqplib (Allow sending log messages to an AMQP server using php-amqplib)
monolog/monolog suggests installing php-console/php-console (Allow sending log messages to Google Chrome)
monolog/monolog suggests installing rollbar/rollbar (Allow sending log messages to Rollbar)
monolog/monolog suggests installing ruflin/elastica (Allow sending log messages to an Elastic Search server)
monolog/monolog suggests installing sentry/sentry (Allow sending log messages to a Sentry server)
Writing lock file
Generating autoload files`

Any tips/suggestios appreciated

Regards,
